modules/guardrails-orchestrator-configuring-regex-guardrails-gateway.adoc

```diff
@@ -104,16 +104,16 @@ kind: GuardrailsOrchestrator
 metadata:
   name: gorch-sample
 spec:
-  orchestratorConfig: "fms-orchestr8-config-nlp"   <1>
-  "gorch-regex-gateway-image-config"                  <2>
-  "fms-orchestr8-config-gateway"                   <3>
+  orchestratorConfig: "fms-orchestr8-config-nlp"   
+  enableBuiltInDetectors: True  <1>
+  enableGuardrailsGateway: True  <2>
+  guardrailsGatewayConfig: "fms-orchestr8-config-gateway" <3>
   replicas: 1
 ----
-<1> The `ConfigMap` for general Guardrails Orchestrator deployment. 
-<2> The `ConfigMap` that sets `chat_generation.service.hostname` and `detectors.hap.service.hostname`.
-<3> The `ConfigMap` that specifies the `detectors` and `routes`.
-
-
+<1> The `enableBuiltInDetectors`, if set to `True`, injects regex detectors as a sidecar container into the orchestrator pod.
+<2> The `enableGuardrailsGateway`, if set to `True`, injects the guardrails gateway as a sidecar container into the orchestrator pod.
+<3> The `guardrailsGatewayConfig` field specifies a `ConfigMap` that reroutes the orchestrator and regex detector routes to specific paths.
+	
 . Deploy the orchestrator custom resource. This creates a service account, deployment, service, and route object in your namespace.
```

Comment:
Insert a noun after "This". Perhaps "This step" or "This action" or "This command"?

See the _IBM Style_ **Pronouns** topic, "Ambiguous pronoun references" section.